AdyenNetworking/APIClient/APIClient.swift

```diff
@@ -137,24 +144,27 @@ public final class APIClient: APIClientProtocol {
     }
     
     /// :nodoc:
-    private lazy var urlSession: URLSession = {
+    private let urlSession: URLSession
```

Comment:
maybe move this line to the top part of the class with the rest of private fields?